Eather009 master

eather009 opened the pull request:
Conflict resolved 
I have updated @amims71 code too. I have updated the input handler to manage blank input with Default data provider and default current city as location

eather009 committed:
1. Get current Location from IP
2. Set Temperature as Integer instead of Decimal
3. arg checking update to ignore empty value for Provider or Location

eather009 committed:
--p and -p both support is enabled. Because dotnet run -p gives error as Project name on mac

eather009 committed:
Conflict resolved

1. Get current Location from IP
2. Set Temperature as Integer instead of Decimal
3. arg checking update to ignore empty value for Provider or Location
4. --p and -p both support is enabled. Because, dotnet run -p gives error as Project name on mac

eather009 committed:
Merge branch 'master' into eather009-master

* master:
  Conflict resolved
  input handler for input from console or cli argument

# Conflicts:
#	nahid/xWeather/Drivers/VisualCrossing/VisualCrossingDriver.cs
#	nahid/xWeather/Program.cs
#	nahid/xWeather/WeatherManager.cs

amims71 reviewed (commented):
What about adding a new input handler that will implement the same interface or extend some existing handler. But will take the location from ip address? So, SOLID will remain there.

eather009 committed:
WeatherApi Location Property added
WeatherApi Time data added
Input handler factory with Default value from IP is added using ipinfo.io api
Show time along with Location in console output

eather009 commented:
I have added a new Pull request with the new update.
